apps/extension/src/ui/domains/Ramps/ramp/useRampTokens.ts

```diff
@@ -0,0 +1,46 @@
+import { keepPreviousData, useQuery } from "@tanstack/react-query"
+import { log } from "extension-shared"
+import { useMemo } from "react"
+
+import { RampsMode } from "../shared/types"
+import { getRampApiUrl } from "./getRampApiUrl"
+import { RampHostAssetsConfig } from "./types"
+import { useRampCurrencies } from "./useRampCurrencies"
+
+// note: currencyCode must be upper case
+const fetchRampAssets = async (
+  currencyCode: string,
+  mode: RampsMode,
+): Promise<RampHostAssetsConfig> => {
```

Comment:
Maybe this should be a `/** */` comment, so it shows up in vscode whenever someone is using this function?

```suggestion
const fetchRampAssets = async (
  /** NOTE: currencyCode must be upper case */
  currencyCode: string,
  mode: RampsMode,
): Promise<RampHostAssetsConfig> => {
```